data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1408,7 +1414,11 @@ descriptions:new_tribe {
             -- TRANSLATORS: Purpose helptext for an atlantean warehouse: Port
             purpose = pgettext("atlanteans_building", "Serves as a base for overseas colonization and trade. Also stores your soldiers, wares and tools."),
             -- TRANSLATORS: Note helptext for an atlantean warehouse: Port
-            note = pgettext("atlanteans_building", "Similar to the Headquarters a Port can be attacked and destroyed by an enemy. It is recommendable to send soldiers to defend it.")
+            note = pgettext("atlanteans_building", "Similar to the Headquarters a Port can be attacked and destroyed by an enemy. It is recommendable to send soldiers to defend it."),
+            -- TRANSLATORS: Note lore for an atlantean production site: Port
+            lore = pgettext("atlanteans_building", "This is the building where expeditions will be equipped to boldly go and find new shores."),
+            -- TRANSLATORS: Note lore_autor for an atlantean production site: Port
+            lore_author = pgettext("atlanteans_building", "Priest of Lutas blessing a new Port")
```

Comment:
I think we can't have Lutas and Satul in the lore as they are worst enemies and Atlantean history is completely divided in Old (Lutas) times and new (Satul) times. As we are in new times I believe any priest of Lutas has been drowned with the Island as it was Lutas will. 
You may cite Ostur the inventor of the new Atlantean ship type that allows for long endurance expeditions

Reply:
Since Satul the Fire God is the worst enemy of Lutas the Sea God I think Satul's priests wouldn't bless a port. I imagined this quote to stem from the time before the Atlanteans fled Atlantis, when Lutas was still their primary deity. But Ostur would also be fine.

Reply:
Oops looks like I missed some aspect of the Atlantean gods.